content/en/docs/concepts/cluster-administration/networking.md

```diff
@@ -258,8 +258,6 @@ Multus supports all [reference plugins](https://github.com/containernetworking/p
 
 [VMware NSX-T](https://docs.vmware.com/en/VMware-NSX-T/index.html) is a network virtualization and security platform. NSX-T can provide network virtualization for a multi-cloud and multi-hypervisor environment and is focused on emerging application frameworks and architectures that have heterogeneous endpoints and technology stacks. In addition to vSphere hypervisors, these environments include other hypervisors such as KVM, containers, and bare metal.
```

Comment:
How about:
```suggestion
[VMware NSX-T](https://docs.vmware.com/en/VMware-NSX-T/index.html) is a network virtualization and security platform. NSX-T can provide network virtualization for cloud or bare-metal infrastructures.
```

Reply:
Does it make sense to have this paragraph if there is nothing else in the entry?

Reply:
Hyperlinking to the specific page that mentions a CNI plugin [feels reasonable to me](https://github.com/kubernetes/website/pull/30093/files#r729784622).

Reply:
The comment you're replying to links to my [previous suggestion](https://github.com/kubernetes/website/pull/30093/files#r729784622) @ixodie, videlicet:
- link to https://docs.vmware.com/en/VMware-NSX-T-Data-Center/2.4/com.vmware.nsxt.ncp_kubernetes.doc/GUID-52A92986-0FDF-43A5-A7BB-C037889F7559.html

I still recommend that; I'm also open to considering other URIs.